### PR TITLE
Clarify environment variable order of precedence

### LINKED ISSUE #5371: Fix note on how environment variables definition in the docs work
This was raised on stack overflow: https://stackoverflow.com/questions/71845565/precedence-of-setting-environment-variables. 

It looks like the behaviour is the opposite of what is documented.

### PR DESCRIPTION
**Pull request by rjmolyneaux**
This PR addresses and will close #5371 ;

### CODE REVIEW

**Comment by luabud** [comment]
This looks good to me! @karthiknadig  would you mind taking a look as well?

File: docs/python/environments.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ MYPROJECT_DBPASSWORD=kKKfa98*11@
 
 You can then set the `python.envFile` setting to `${workspaceFolder}/prod.env`, then set the `envFile` property in the debug configuration to `${workspaceFolder}/dev.env`.
 
-> **Note**: When environment variables are specified using multiple methods, be aware that there is an order of precedence. Environment variables contained in the `.env` file specified by the `python.envFile` setting (user or workspace) will override variables defined in the `envFile` specified in `launch.json`, as well as any `env` variables defined in the `launch.json` file itself. Similarly, environment variables defined in the `envFile` specified in `launch.json` will override `env` variables defined in the `launch.json` file.
+> **Note**: When environment variables are specified using multiple methods, be aware that there is an order of precedence. All `env` variables defined in the `launch.json` file will override variables contained in the `.env` file, specified by the `python.envFile` setting (user or workspace).Similarly, `env` variables defined in the `launch.json` file will override the environment variables defined in the `envFile` that are specified in `launch.json`.
```
Comment (karthiknadig) [inline docs/python/environments.md:289]:
```suggestion
> **Note**: When environment variables are specified using multiple methods, be aware that there is an order of precedence. All `env` variables defined in the `launch.json` file will override variables contained in the `.env` file, specified by the `python.envFile` setting (user or workspace). Similarly, `env` variables defined in the `launch.json` file will override the environment variables defined in the `envFile` that are specified in `launch.json`.
```

**Review by karthiknadig — APPROVED** [review]

**Review by rjmolyneaux — COMMENTED** [review]
Thanks for the updates @karthiknadig and @luabud!

File: docs/python/environments.md (hunk 1 above)
Comment (luabud) [reply docs/python/environments.md]: oh, great catch!!

**Pull request merged**